src/services/index.ts

```diff
@@ -34,13 +102,50 @@ export const block = {
 };
 
 export const swork = {
-  register: (req: Request, res: Response, next: any) => {
+  register: (req: Request, res: Response, next: NextFunction) => {
+    api.isReady.then(async api => {
```

Comment:
looks like the api.isReady.then... is a common block. A function could be extracted like:
```javascript
async function withApi(fn, next) {
 const api2 = await api.isReady;
  try {
    await fn(api2);
  } catch(ex) {
  next(ex);
  }
}

// usage:
withApi(async (api) => {
  const krp = loadKeyringPair(req);
  res.json(await register(api, krp, req); // the res.json part could be placed in the withApi function if necessary.
});
```